engine/servermaster/jobmanager.go

```diff
@@ -313,6 +385,23 @@ func (jm *JobManagerImpl) ListJobs(ctx context.Context, req *pb.ListJobsRequest)
 	return resp, nil
 }
 
+func buildPBJobWithStatus(masterMeta *frameModel.MasterMetaKVData, status *pb.Status, errJob error) (*pb.Job, error) {
```

Comment:
Please comment on what kind of errors `errJob` should be, and how it is used to build a `pb.Job`.

Reply:
Out of date.